extensions/default/src/Panels/StudyBrowser/PanelStudyBrowser.tsx

```diff
@@ -10,43 +10,38 @@ import { defaultActionIcons, defaultViewPresets } from './constants';
 import {
   DropdownMenu,
   DropdownMenuContent,
-  DropdownMenuItem,
   DropdownMenuTrigger,
   Icons,
   Button,
 } from '@ohif/ui-next';
 
 const { sortStudyInstances, formatDate, createStudyBrowserTabs } = utils;
 
-const StudyMenuItems = ({ StudyInstanceUID, commandsManager }) => {
+const getMenuItems = ({ commandsManager, items, servicesManager, ...props }) => {
+  const { customizationService } = servicesManager.services;
+
+  // Todo: this feels odd, but maybe my feelings are wrong
+  const menuContent = customizationService.getCustomization('ohif.menuContent');
```

Comment:
I think this is generally a good idea, but the menu content should be named something like ohif.menuContent.PanelStudyBrowser  and the name should come from a parameter so that the panel props can change the entire menu for an instance, or just change the menu directly.